### ISSUE TITLE
maintenance of NWB library

### ISSUE THREAD

**Issue by esakal** [insin/nwb#537]
This issue is a: 
- Other

Hello!
NWB is such an important library those days where the devop changes frequently and become a complicated yet essential part of the development. I loved that you covered many technology stacks (web, preact, react....) and that you abstract the implementation details yet support the common formats (es modules, umd, lib). 

I wanted to ask about the maintenance status of this library as it is been a while since you released new version or pushed new commits, there are deprecation and security warnings on libraries that are used by nwb and issues are not getting attentions. 

This is not a criticism, I know how hard it is to try and be on top of all those issues/updates etc. But I would appreciate your answer about the near future planned effort on this library. When are you planning to fix all those deprecations / security issues and release new version?

Thanks!
Eran.

**Comment by samrith-s** [comment]
I agree with this. I would love to help maintain NWB, especially in rolling out features and improvements relevant to the current state of the JS ecosystem. Also, to get rid of these humongous list of warnings:
![image](https://user-images.githubusercontent.com/9032162/62006494-6eb72e00-b141-11e9-8b5f-c03521fca391.png)

**Comment by bdaenen** [comment]
@insin 
I'd like some info on the maintenance status of this repository as well. We are about to start developing a relatively large component library using NWB, and generally the initial setup with NWB was a pretty much perfect experience.

The lack of recent activity/updates is the only thing that concerns me. I wouldn't mind helping out with maintenance but considering there haven't been any merged pull requests since 2018 I'd rather not invest any time before understanding if this project is still going anywhere.

**Comment by tnrich** [comment]
@insin seems like he hasn't been on github for a couple weeks. Maybe he'll show up soon? I think if we don't get a response within another month it is time to either fork and release a new version of nwb maintained by different people or .. abandon ship for another similar dev tool. I've loved using nwb for the past couple years and found it has greatly eased development, but if it isn't getting maintained then something will have to be done.

**Comment by stevensacks** [comment]
Speaking of abandoned, does anyone know how to fix the issue where import autocompletes the `/es/` path and not the root export? I opened an issue in May and it's October. I feel like maybe it's a simple configuration change but I can't figure it out.

https://github.com/insin/nwb/issues/530

**Comment by Gytree** [comment]
any news ?

**Comment by samrith-s** [comment]
As of right now, I have found the following up-to-date alternatives; at least for the React ecosystem:
- [Create React App](https://github.com/facebook/create-react-app)
- [Create React Library](https://github.com/udilia/create-react-library)

**Comment by tylim88** [comment]
has issue with downloang phantomjs  v2.1.1

I use https://neutrinojs.org/ as alternative

**Comment by tnrich** [comment]
@insin make some of us maintainers? We'd love to help maintain this repo!

**Comment by mehdisadeghi** [comment]
Just migrated my react component to neutrinojs. Works like charm.

**Comment by insin** [comment]
Hi @esakal, I haven't been using React at work for a couple of years now, which was one of the main things which allowed me to keep nwb updated (as I had to keep it working, and any improvements were benefits to my work), and have had limited time for working on my own projects and open source stuff in general over the last year.

I've had a bit of time this week and I'm surprised at how relatively static things have stayed over the last year and a half - Webpack 4 is still the stable version, as is Babel 7, which I'd already completed 95% of the work to migrate to. There's now also an official solution for React hot reloading, which originally needed a really hacky approach for Babel 7 which stopped me from being happy with merging that branch.

Dependencies are now up to date in `master` and the Babel 7 branch has been merged, so there'll be a new release soon.
